content/en/real_user_monitoring/reactnative/integrated_libraries.md

```diff
@@ -0,0 +1,79 @@
+---
+title: React Native Integrated Libraries
+kind: faq
+description: Collect RUM data from your React Native projects.
+further_reading:
+- link: https://github.com/DataDog/dd-sdk-reactnative
+  tag: GitHub
+  text: dd-sdk-reactnative Source code
+- link: https://www.datadoghq.com/blog/react-native-monitoring/
+  tag: Blog
+  text: Monitor your React Native applications
+- link: real_user_monitoring/explorer/
+  tag: Documentation
+  text: Learn how to explore your RUM data
+
+---
+## Setup
+
+**Note**: This package is an integration for `react-navigation` library, please make sure you first install and setup the core `mobile-react-native` SDK.
+
+To install with NPM, run:
+
+```sh
+npm install @datadog/mobile-react-navigation
+```
+
+To install with Yarn, run:
+
+```sh
+yarn add @datadog/mobile-react-navigation
+```
+
+### Track view navigation
+To track changes in navigation as RUM Views, set the `onready` callback of your `NavigationContainer` component as follow. You can use the optional `ViewNamePredicate` parameter to replace the automatically detected View name with something more relevant to your use case.
+
+Returning `null` in the `ViewNamePredicate` prevents the new RUM View from being created. The previous RUM View remains active.
+
+```js
+import * as React from 'react';
+import { DdRumReactNavigationTracking, ViewNamePredicate } from '@datadog/mobile-react-navigation';
+import { Route } from "@react-navigation/native";
+
+const viewNamePredicate: ViewNamePredicate = function customViewNamePredicate(route: Route<string, any | undefined>, trackedName: string) {
+  return "My custom View Name"
+}
+
+function App() {
+  const navigationRef = React.useRef(null);
+  return (
+    <View>
+      <NavigationContainer ref={navigationRef} onReady={() => {
+        DdRumReactNavigationTracking.startTrackingViews(navigationRef.current, viewNamePredicate)
+      }}>
+        // …
+      </NavigationContainer>
+    </View>
+  );
+}
+```
+
+**Note**: Only one `NavigationContainer` can be tracked at the time. If you need to track another container, stop tracking the previous one first, using `DdRumReactNavigationTracking.stopTrackingViews()`.
```

Comment:
This part is related to the React Navigation integration, I think it's not obvious if we put it at the start of this page. I think it would make more sense to have it under the `### Integrate with React Navigation` section, in replacement to the current content of that section.

Reply:
Hi @louiszawadzki, I added a section for React Navigation and React Native Integration. I left the RUM Integrations section, but not sure if this is what you had in mind. Please feel free to **Add the suggestion** so that I can accept. Thanks again for your guidance here!